ui/recieve_page.go

```diff
@@ -1 +1,301 @@
+												win.outputs.selectedAccountBalanceLabel.Layout(win.gtx)
+											})
+										}),
+									)
+								}),
+								layout.Rigid(func() {
+									layout.Inset{Left: unit.Dp(20)}.Layout(win.gtx, func() {
+										layout.Flex{}.Layout(win.gtx,
+											layout.Rigid(func() {
+												layout.Inset{Bottom: unit.Dp(5)}.Layout(win.gtx, func() {
+													win.outputs.selectedWalletNameLabel.Layout(win.gtx)
+												})
+											}),
+											layout.Rigid(func() {
+												layout.Inset{Left: unit.Dp(22)}.Layout(win.gtx, func() {
+													win.outputs.selectedWalletBalLabel.Layout(win.gtx)
+												})
+											}),
+										)
+									})
+								}),
+							)
+						})
+					}
+					decredmaterial.Card{}.Layout(win.gtx, selectedDetails)
+				}),
+			)
+		}),
+	)
+}
+
+func (win *Window) qrCodeAddressColumn() {
+	qrCode, err := qrcode.New(win.addrs, qrcode.Highest)
+	if err != nil {
+		win.outputs.err.Text = err.Error()
+		return
+	}
+
+	qrCode.DisableBorder = true
+	layout.Flex{Axis: layout.Vertical}.Layout(win.gtx,
+		layout.Rigid(func() {
+			layout.Inset{Top: unit.Dp(16)}.Layout(win.gtx, func() {
+				layout.Flex{}.Layout(win.gtx,
+					layout.Flexed(0.2, func() {
+					}),
+					layout.Flexed(1, func() {
+						img := win.theme.Image(paint.NewImageOp(qrCode.Image(520)))
+						img.Src.Rect.Max.X = 521
+						img.Src.Rect.Max.Y = 521
+						img.Scale = 0.5
+						img.Layout(win.gtx)
+					}),
+				)
+			})
+		}),
+		layout.Rigid(func() {
+			layout.Inset{Top: unit.Dp(16)}.Layout(win.gtx, func() {
+				layout.Flex{}.Layout(win.gtx,
+					layout.Flexed(0.1, func() {
+					}),
+					layout.Flexed(1, func() {
+						win.receiveAddressColumn()
+					}),
+				)
+			})
+		}),
+	)
+}
+
+func (win *Window) receiveAddressColumn() {
+	layout.Flex{}.Layout(win.gtx,
+		layout.Rigid(func() {
+			win.outputs.receiveAddressLabel.Text = win.addrs
+			win.outputs.receiveAddressLabel.Layout(win.gtx)
+		}),
+		layout.Rigid(func() {
+			layout.Inset{Left: unit.Dp(16)}.Layout(win.gtx, func() {
+				for win.inputs.receiveIcons.copy.Clicked(win.gtx) {
+					clipboard.WriteAll(win.addrs)
+					win.addressCopiedLabel.Text = "Address Copied"
+					time.AfterFunc(time.Second*9, func() {
+						win.addressCopiedLabel.Text = ""
+					})
+				}
+				win.outputs.copy.Layout(win.gtx, &win.inputs.receiveIcons.copy)
+			})
+		}),
+	)
+}
+
+func (win *Window) setDefaultPageValues() {
+	info := win.walletInfo.Wallets[win.selected]
+
+	for i := range info.Accounts {
+		if len(info.Accounts) == 0 {
+			continue
+		}
+
+		for win.inputs.receiveIcons.newAddress.Clicked(win.gtx) {
+			addr, err := win.wallet.NextAddress(info.ID, info.Accounts[i].Number)
+			if err != nil {
+				win.outputs.err.Text = err.Error()
+				return
+			}
+			info.Accounts[i].CurrentAddress = addr
+			win.isNewAddrModal = false
+		}
+
+		win.setSelectedAccount(info, info.Accounts[i])
+		break
+	}
+}
```

Comment:
Unnecessary

Reply:
You may need to be more explicit.

Reply:
I'm saying the whole function is really unnecessary. What does default even mean in this context? why does it need to be set each time the page is laid out?. And clicks should be handled in `handle_input.go`